src/common/util.cpp

```diff
@@ -1081,4 +1085,38 @@ std::string get_nix_version_display_string()
       return std::to_string((uint64_t)(ts / (3600 * 24 * 30.5))) + tr(" months");
     return tr("a long time");
   }
+
+  std::string get_human_readable_bytes(uint64_t bytes)
+  {
+    // Use 1024 for "kilo", 1024*1024 for "mega" and so on instead of the more modern and standard-conforming
```

Comment:
I'm not sure if I agree with this.  Changing one thing to be wrong so it conforms with other things that are wrong seems...wrong.  That said, I don't care that much and there is merit to being consistent with upstream.

Reply:
I'm in agreement with @tewinget on this one.  I *hate* that the block explorer shows this because of this 1024 division:

> TX Size: 3.2158 kB

(I know it's a separate program, but it's the same basic issue).

> there is merit to being consistent with upstream.

In this case I don't think there's a lot of merit: we're only talking about a string designed for human consumption, so it seems unlikely that  base-10 change here is going to result in conflicts down the line—and that to me is the main merit in staying consistent with upstream.

Reply:
I've changed to use standard sizes, where 1000 bytes = 1kb.